### PR TITLE
[Falco] Add the option to disable async CRI metadata fetching

### PR DESCRIPTION
**Pull request by Annegies**
/kind feature

/kind chart-release

**Any specific area of the project related to this PR?**
/area falco-chart

**What this PR does / why we need it**:
Enables us to use the `--disable-cri-async`. 

**Which issue(s) this PR fixes**:
None

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

File: falco/values.yaml [hunk 1]
```diff
@@ -177,6 +177,13 @@ falco:
   # for up to 2 seconds without reaching the timeout.
 
   output_timeout: 2000
+  # Disable asynchronous CRI metadata fetching.
+  # This is useful to let the input event wait for the container metadata fetch
+  # to finish before moving forward. Async fetching, in some environments leads
+  # to empty fields for container metadata when the fetch is not fast enough to be
+  # completed asynchronously. This can have a performance penalty on your environment
+  # depending on the number of containers and the frequency at which they are created/started/stopped
+  disableCriAsync: false
```
Comment (cpanato) [inline falco/values.yaml:186]: nit: should be `disableCRIAsync`

Comment (dza89) [reply falco/values.yaml]: Two letter abbreviations is both capital letters.
From three and up it's only the first in capital.

Couldn't find a good source but XMLHTTPRequest or xmlHttpRequest

Comment (Annegies) [reply falco/values.yaml]: fair, renamed!

**Comment by Annegies** [comment]
resolved comments

**Comment by Annegies** [comment]
is this okay?

**Review by leogr — COMMENTED** [review]
Hi @Annegies 

Thank you for your effort!

Although it seems good to me, however, adding that flag was already possible by using `extraArgs`:
https://github.com/falcosecurity/charts/blob/master/falco/templates/daemonset.yaml#L72

I'm wondering if there's a real benefit to introduce an explicit configuration value for that (which could be redundant).

Anyhow, I have no strong opinion. It will be ok for me if this PR gets approved.
Just awaiting a bit more, to hear your and @falcosecurity/charts-maintainers opinions.

**Comment by Annegies** [comment]
Hi @leogr!

Completely missed that value.
in that case imo this PR is completely redundant, so i'll close it.

Thank you for pointing this out!

**Pull request closed without merging**